webapp/loggedin/modules/analysis/calculation/actions/state.js

```diff
@@ -5,11 +5,10 @@ import * as NodeDef from '@core/survey/nodeDef'
 import * as Step from '@common/analysis/processingStep'
 import * as Calculation from '@common/analysis/processingStepCalculation'
 
-import * as SurveyState from '@webapp/survey/surveyState'
+import { SurveyState, NodeDefsActions } from '@webapp/store/survey'
```

Comment:
It looks weird to me?
What is the relation between Survey and NodeDef?

Reply:
NodeDefs, Categories, Taxonomies are all survey metadata . Accessed through Survey . We can talk about it later